components/work_templates/components/menu.tsx

```diff
@@ -63,6 +64,10 @@ interface MenuProps {
 const Menu = ({className, disableQuickUse, categories, workTemplates, currentCategoryId, changeCategory, onTemplateSelected}: MenuProps) => {
     const {formatMessage} = useIntl();
 
+    useEffect(() => {
+        trackEvent('work_templates', 'pageview_menu');
```

Comment:
I'd suggest to put the category in a constant value https://github.com/mattermost/mattermost-webapp/blob/master/utils/constants.tsx#L729